app/src/main/res/layout/fragment_adminpanel_dailymarket.xml

```diff
@@ -0,0 +1,307 @@
+            app:layout_constraintEnd_toStartOf="@+id/openMarketButton"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+            <Button
+                android:id="@+id/openMarketButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="24dp"
+                android:background="@drawable/black_white_rounded_button"
+                android:padding="4dp"
+                android:text="Open Market"
+                android:textColor="@color/neon_green"
+                app:layout_constraintBottom_toTopOf="@id/closeMarketButton"
+                app:layout_constraintStart_toEndOf="@id/stockIdEditText"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <Button
+                android:id="@+id/closeMarketButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginEnd="20dp"
+                android:layout_marginTop="10dp"
+                android:background="@drawable/black_white_rounded_button"
+                android:padding="4dp"
+                android:text="Close Market"
+                android:textColor="@color/neon_yellow"
+                app:layout_constraintEnd_toEndOf="@id/openMarketButton"
+                app:layout_constraintHorizontal_bias="0.0"
+                app:layout_constraintStart_toStartOf="@id/openMarketButton"
+                app:layout_constraintTop_toBottomOf="@id/openMarketButton" />
+
+
+            <View
+                android:id="@+id/view3"
+                android:layout_width="match_parent"
+                android:layout_height="2dp"
+                android:layout_margin="5dp"
+                android:background="@color/black"
+                app:layout_constraintTop_toBottomOf="@+id/closeMarketButton"
+                tools:layout_editor_absoluteX="8dp" />
+
+            <TextView
+                android:id="@+id/stockIdEditText2"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:backgroundTint="@color/neutral_font_color"
+                android:inputType="number"
+                android:text="Daily LeaderBoard"
+                android:textColor="@color/neutral_font_color"
+                android:textColorHint="@color/neutral_font_color"
+                app:layout_constraintBottom_toBottomOf="@+id/updateEndOFDayValue"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="@+id/updateEndOFDayValue" />
+
+            <Button
+                android:id="@+id/updateEndOFDayValue"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="16dp"
```

Comment:
remove hard coded paddings and margins everywhere in this xml layout file !

use dimens.xml for this